adafruit_ltr390.py

```diff
@@ -91,7 +101,10 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(
+        cls,
+        value_tuples: Tuple[str, int, str, Optional[float], int, Optional[float]],
```

Comment:
This should be wrapped in `Iterable` from the `typing` module, see the previous review if you need the syntax.